src/Microsoft.HttpRepl.IntegrationTests/Mocks/MockDirectoryStructure.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Mocks
+{
+    internal class MockDirectoryStructure : IDirectoryStructure
```

Comment:
I'm not convince the Fake is necessary here, since `DirectoryStructure` is so lightweight as it is. It does keep the calls to it in the tests cleaner, and it's not what you're testing against, so it's probably ok. Just wanted to throw it out there.

Reply:
That's true. Removed it here:
https://github.com/aspnet/HttpRepl/commit/7b4b20697f18184252ddd9c3a5ecd278de1d5c55

Thanks!